job_definitions/functional_tests_legacy.yml

```diff
@@ -85,6 +85,8 @@
           mkdir -p reports
           rm -f screenshot*
 
+          ./scripts/create_users.sh example
```

Comment:
I assume it doesn't matter that the environment variables that get set during the `create_users` script include some bonus ones not defined above in this jenkins job?
https://github.com/alphagov/digitalmarketplace-functional-tests/blob/jenkins/scripts/envs/example.sh

Reply:
Yeah I've had a look and they're harmless.